doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/localization/TextLocalizationPage.swift

```diff
@@ -0,0 +1,36 @@
+//
+
+import SwiftUI
+
+// @maatteogekko: I'm copying these initializers from my previous
```

Comment:
Let's not commit personal comments into the code base. Please use Discord, email, GitHub comments, etc. to communicate with other team members.